x-pack/plugins/lens/public/indexpattern_datasource/change_indexpattern.tsx

```diff
@@ -1,93 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
```

Comment:
ℹ️ I am deleting this file because it is not used anymore in Lens. This must be a leftover from the phase 1. I can create a sepatate PR for the removal but as I am dealing here with the dataview picker lists I think I can also remove it here :)